README.md

```diff
@@ -9,7 +9,7 @@ This is continuation of venerable JA2-Stracciatella project.
 
 The goal of the project is to make Jagged Alliance 2 available on a wide range of platforms, improve its stability, fix bugs and provide a stable platform for mod development.  At the moment the goal is mostly to fix bugs.
 
-Official Homepage: http://ja2-stracciatella.github.io/
+[Official Homepage](http://ja2-stracciatella.github.io)
```

Comment:
This is not good for two reasons: the URL is not visible any more and the line stands out. The intent is to lay it out, so if the linter is complaining about the bare URL, just add the (reduntant) URL syntax around it with itself as the text.

Reply:
I've reverted the change.